src/signals/IncidentMap/components/IncidentMap/IncidentMap.tsx

```diff
@@ -60,6 +72,8 @@ export const IncidentMap = () => {
       <StyledMap
         data-testid="incidentMap"
         fullScreen={false}
+        setInstance={setMap}
+        hasGPSControl
```

Comment:
Denk niet dat dit een functie heeft. Werd voorheen gebruikt om de GPS button conditioneel te renderen, maar heeft zijn functie verloren denk ik. (https://github.com/Amsterdam/signals-frontend/pull/1003/files)

Reply:
Removed

Reply:
On second thought (and after tests that were failing), map.flyTo needs to have a map. So the map needs to be set.         setInstance={setMap} has been added to the code again.

Reply:
True! I just meant the `hasGPSControl` sorry.